connector/src/main/java/org/geysermc/connector/network/translators/world/block/entity/SkullBlockEntityTranslator.java

```diff
@@ -68,4 +88,119 @@ public CompoundTag getDefaultBedrockTag(String bedrockId, int x, int y, int z) {
         tagBuilder.byteTag("SkullType", (byte) 0);
         return tagBuilder.buildRootTag();
     }
+
+    public static SerializedSkin getSkin(com.github.steveice10.opennbt.tag.builtin.CompoundTag tag) {
+        if (tag.contains("Owner") && !CACHED_SKULLS.containsKey(tag)) {
+            com.github.steveice10.opennbt.tag.builtin.CompoundTag Owner = tag.get("Owner");
+            com.github.steveice10.opennbt.tag.builtin.CompoundTag Properties = Owner.get("Properties");
+            ListTag Textures = Properties.get("textures");
+            LinkedHashMap tag1 = (LinkedHashMap) Textures.get(0).getValue();
+            StringTag texture = (StringTag) tag1.get("Value");
+            byte[] decoded = Base64.getDecoder().decode(texture.getValue().getBytes());
+            String url = new String(decoded);
+
+            ObjectMapper objectMapper = new ObjectMapper();
+            try {
+                JsonNode jsonNodeRoot = objectMapper.readTree(url);
+                JsonNode jsonNodeUrl = jsonNodeRoot.get("textures").get("SKIN").get("url");
+                CompletableFuture<SkinProvider.Skin> skinCompletableFuture = SkinProvider.requestSkin(UUID.randomUUID(), jsonNodeUrl.asText(), false);
+                SkinProvider.Skin skin = skinCompletableFuture.get();
+                SerializedSkin serializedSkin = SerializedSkin.of(
+                        "Steve", SkinProvider.SkinGeometry.getSkull().getGeometryName(), ImageData.of(skin.getSkinData()), Collections.emptyList(),
+                        ImageData.EMPTY, SkinProvider.SkinGeometry.getSkull().getGeometryData(), "", true, false, false, "", UUID.randomUUID().toString()
+                );
+                return serializedSkin;
+            } catch (IOException e) {
+                e.printStackTrace();
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            } catch (ExecutionException e) {
+                e.printStackTrace();
+            }

```

Comment:
This should be collapsed to either ` } catch (IOException | InterruptedException | ExecutionException e) {` or just `} catch (Exception e) {`